frappe/handler.py

```diff
@@ -144,8 +144,8 @@ def upload_file():
 	file_url = frappe.form_dict.file_url
 	folder = frappe.form_dict.folder or 'Home'
 	method = frappe.form_dict.method
+	filename = frappe.form_dict.get('file_name')
```

Comment:
This is really a nitpick, but should this be consistent with the rest of the variables?

```suggestion
	filename = frappe.form_dict.file_name
```